.travis.yml

```diff
@@ -97,5 +97,12 @@ matrix:
       - "scripts/build_assembly_no_test.sh scalding-core"
       - "scripts/test_typed_tutorials.sh"
 
+    - scala: 2.10.4
+      env: BUILD="test execution tutorials"
+      script:
+      - "scripts/build_assembly_no_test.sh scalding-core"
```

Comment:
Without build scalding-core, travis gave me this failure:
https://travis-ci.org/twitter/scalding/jobs/50681252